lucene/directory-provider/src/test/resources/testing-flush-loopback.xml

```diff
@@ -28,7 +28,7 @@
 	<pbcast.STABLE stability_delay="1000" desired_avg_gossip="50000"
 		max_bytes="400000" />
 	<pbcast.GMS
-			print_local_addr="true"
+			print_local_addr="false"
```

Comment:
Is this change related?

Reply:
Not really, but it merely aligns it with the behaviour of all other tests. Harmless.